src/QsCompiler/BondSchemas/QirExecutionWrapper/Extensions.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Linq;
+using Microsoft.Quantum.QsCompiler.BondSchemas.EntryPoint;
+
+namespace Microsoft.Quantum.QsCompiler.BondSchemas.QirExecutionWrapper
+{
+    /// <summary>
+    /// This class provides extension methods for objects in the Microsoft.Quantum.QsCompiler.BondSchemas.QirExecutionWrapper namespace.
+    /// </summary>
+    internal static class Extensions
+    {
+        /// <summary>
+        /// Determine whether the values of two object instances are equal.
+        /// </summary>
+        public static bool ValueEquals(this QirExecutionWrapper executionWrapperA, QirExecutionWrapper executionWrapperB)
+        {
+            if (executionWrapperA == null && executionWrapperB == null)
```

Comment:
You can add a `#nullable enable` pragma at the beginning of this source file and you won't need to check for null here which would make the code cleaner.

Reply:
I think this is still not resolved.